fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/IndexBuilderImpl.java

```diff
@@ -77,10 +95,28 @@ public void rebuild() {
                             //we're only interested in sidecar subpaths
                             try {
                                 final var headers = deserializeHeaders(objSession.read(subpath));
-                                fedoraIds.add(headers.getId());
+                                final var fedoraId = headers.getId();
+                                fedoraIds.add(fedoraId);
                                 if (headers.isArchivalGroup()) {
                                     rootId.set(headers.getId());
                                 }
+
+                                if (!fedoraId.equals(rootFedoraId)) {
+                                    var parentId = headers.getParent();
+
+                                    if (parentId == null) {
+                                        if (!fedoraId.endsWith(FCR_METADATA) &&
+                                                !fedoraId.endsWith(FCR_ACL) &&
+                                                !fedoraId.endsWith(FCR_VERSIONS)) {
```

Comment:
I don't think TimeMaps get persisted to OCFL.

Reply:
Yes that's true. I'll pull that out.